components/centos/centos-k8s-singlenode-setup/README.rst

```diff
@@ -12,7 +12,7 @@ QuickStart
 
 2. Create a directory for the Vagrant Box
 
-  ``$ mkdir directory && cd directory``
+  ``$ mkdir directory && cd $_``
```

Comment:
@ryanmaclean do you want to update the PR or I can close this PR and you can send a new PR?